packages/one-app-bundler/package.json

```diff
@@ -75,24 +75,6 @@
     "webpack-subresource-integrity": "^1.3.4",
     "yargs": "^12.0.5"
   },
-  "devDependencies": {
-    "@americanexpress/one-app-ducks": "*",
-    "@americanexpress/one-app-router": "*",
-    "core-js": "^3.2.1",
-    "holocron": "*",
```

Comment:
Are these dev deps used in any other package in this monorepo? I thought the practice was to only put the "common" dev deps in the top level package.json file but these devDeps are only used in this package I thought...

Reply:
Thank you for the feedback, i'll update this to have only shared dev dependencies.
https://github.com/lerna/lerna#common-devdependencies

Reply:
updated on 6b18512380bbfb1cff491b821b10dbc037e4d1a9